src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/DefaultValueHolder.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.Json.Serialization.Metadata
+{
+    /// <summary>
+    /// Helper class used for calculating the default value for a given System.Type instance.
+    /// </summary>
+    internal sealed class DefaultValueHolder
+    {
+        private DefaultValueHolder(Type type)
+        {
+            if (type.IsValueType && Nullable.GetUnderlyingType(type) == null)
+            {
+                DefaultValue = Activator.CreateInstance(type);
```

Comment:
I think we want to call `RuntimeHelpers.GetUninitializedObject` here instead. See https://github.com/dotnet/runtime/pull/47722 for reasoning.

Reply:
Presumably we'd still need to check if `type` is a struct though right?

Reply:
That method is not available in `netstandard2.0` 😑 I'm guessing it's equivalent to the old `FormatterServices.GetUninitializedObject` but I'd rather not introduce a dependency on `System.Runtime.Serialization`.

Reply:
> Presumably we'd still need to check if type is a struct though right?

Yes.

> That method is not available in netstandard2.0

You can use the same `#if` as I did in #47722.

Reply:
Cool, I'll do that then, given there is precedent 👍